src/test/java/strategy/AdditionInCart.java

```diff
@@ -0,0 +1,22 @@
+package strategy;
+
+import pages.MyAccountPage;
+
+public class AdditionInCart implements Addition{
+
+    @Override
+    public void addItem() {
+
+        MyAccountPage accountPage = new MyAccountPage(false);
+               accountPage.jumpToHomePage()
+                .goToProductByName("Faded Short Sleeve T-shirts")
```

Comment:
it is not a good practice to hardcode test data, youve better place it to test data file, or at least move to test level